app/livechat/client/views/app/livechatAppearance.js

```diff
@@ -213,52 +182,52 @@ Template.livechatAppearance.events({
+		const settingOfflineSuccess = getSettingFromAppearance(instance, 'Livechat_offline_success_message');
 		instance.offlineSuccessMessage.set(settingOfflineSuccess && settingOfflineSuccess.value);
 
-		const settingOfflineTitle = LivechatAppearance.findOne('Livechat_offline_title');
+		const settingOfflineTitle = getSettingFromAppearance(instance, 'Livechat_offline_title');
 		instance.titleOffline.set(settingOfflineTitle && settingOfflineTitle.value);
 
-		const settingOfflineTitleColor = LivechatAppearance.findOne('Livechat_offline_title_color');
+		const settingOfflineTitleColor = getSettingFromAppearance(instance, 'Livechat_offline_title_color');
 		instance.colorOffline.set(settingOfflineTitleColor && settingOfflineTitleColor.value);
 
-		const settingConversationFinishedMessage = LivechatAppearance.findOne('Livechat_conversation_finished_message');
+		const settingConversationFinishedMessage = getSettingFromAppearance(instance, 'Livechat_conversation_finished_message');
 		instance.conversationFinishedMessage.set(settingConversationFinishedMessage && settingConversationFinishedMessage.value);
 
-		const settingConversationFinishedText = LivechatAppearance.findOne('Livechat_conversation_finished_text');
+		const settingConversationFinishedText = getSettingFromAppearance(instance, 'Livechat_conversation_finished_text');
 		instance.conversationFinishedText.set(settingConversationFinishedText && settingConversationFinishedText.value);
 
-		const settingRegistrationFormEnabled = LivechatAppearance.findOne('Livechat_registration_form');
+		const settingRegistrationFormEnabled = getSettingFromAppearance(instance, 'Livechat_registration_form');
 		instance.registrationFormEnabled.set(settingRegistrationFormEnabled && settingRegistrationFormEnabled.value);
 
-		const settingRegistrationFormNameFieldEnabled = LivechatAppearance.findOne('Livechat_name_field_registration_form');
+		const settingRegistrationFormNameFieldEnabled = getSettingFromAppearance(instance, 'Livechat_name_field_registration_form');
 		instance.registrationFormNameFieldEnabled.set(settingRegistrationFormNameFieldEnabled && settingRegistrationFormNameFieldEnabled.value);
 
-		const settingRegistrationFormEmailFieldEnabled = LivechatAppearance.findOne('Livechat_email_field_registration_form');
+		const settingRegistrationFormEmailFieldEnabled = getSettingFromAppearance(instance, 'Livechat_email_field_registration_form');
 		instance.registrationFormEmailFieldEnabled.set(settingRegistrationFormEmailFieldEnabled && settingRegistrationFormEmailFieldEnabled.value);
 
-		const settingRegistrationFormMessage = LivechatAppearance.findOne('Livechat_registration_form_message');
+		const settingRegistrationFormMessage = getSettingFromAppearance(instance, 'Livechat_registration_form_message');
 		instance.registrationFormMessage.set(settingRegistrationFormMessage && settingRegistrationFormMessage.value);
 	},
 	'submit .rocket-form'(e, instance) {
```

Comment:
When you submit the new values and the process goes well, you need to reload and set the appearance, like this:
https://github.com/RocketChat/Rocket.Chat/pull/15510/files#diff-5c375a4ce9bfb78e354776f4b65a7270R131

Otherwise, in case the user clicks on the `reset` button, the settings will be reset with the oldest data.